meta-lmp-base/recipes-bsp/efitools/efitools_git.bb

```diff
@@ -45,14 +45,28 @@ python do_prepare_local_auths() {
     import shutil
     import os
 
-    # Use auths already generated by the user
+    # Generate noPK.auth and noKEK.auth if they do not exist
     for _ in ('PK', 'KEK', 'DB', 'DBX', 'noPK', 'noKEK'):
         file = _ + '.auth'
-        src = dir + file
+        src = os.path.join(dir, file)
         if not os.path.isfile(src):
-            bb.fatal("File '%s' not found!" % src)
-        shutil.copyfile(src, d.expand('${S}/') + file)
-
+            if _ in ('noPK', 'noKEK'):
+                bb.note("Generating '%s' in WORKDIR" % file)
```

Comment:
I don't know is it's better to use a _warning_ there instead of `note` otherwise it will go unnoticed during the build and only be noticeable looking at the entire build log which is not usually done.

Reply:
right, I chose a note because IMO users should know what keys they have.